charts/kubewarden-controller/Chart.yaml

```
@@ -42,7 +42,7 @@ annotations:
   catalog.cattle.io/requests-cpu: "250m"
   catalog.cattle.io/requests-memory: "50Mi"
   catalog.cattle.io/rancher-version: ">= 2.6.0-0 <= 2.8.100-0" # Chart will only be available for users in the specified Rancher version(s), here its 2.5.0-2.5.99. This _must_ use build metadata or it won't work correctly for future RC's.
-  catalog.cattle.io/upstream-version: 2.0.2
+  catalog.cattle.io/upstream-version: 2.0.3-rc1
```

```suggestion
  catalog.cattle.io/upstream-version: 2.1.0-rc1
```

I think this should be a minor bump of `version`